frontend/src/components/VAudioTrack/VAudioTrack.vue

```diff
@@ -470,7 +470,10 @@ export default defineComponent({
     )
     const ariaLabel = computed(() =>
       isComposite.value
-        ? i18n.t("audio-track.aria-label-interactive", {
+        ? // TODO: Confirm with @dhruvk when this _shouldn't_ be the "seekable" variant
+          // `seekable` variant for the string wasn't used before and changing this makes
+          // non-seekable variant unused: can we remove one or the other and just use one?
+          i18n.t("audio-track.aria-label-interactive-seekable", {
```

Comment:
@dhruvkb Can you confirm what the correct label should be here? I haven't gone digging for when these labels were introduced or if the seekable one _used_ to be used but was accidentally removed at some point. We don't use both, so we can remove the one we don't use if this change doesn't require additional work to choose the correct one.

In what cases is the player interactive but not seekable? At first I thought it might be the box layout (used for all results grid) but that one is seekable when I tested :thinking:

Reply:
I recall the box layout _not_ being seekable previously. It isn't via mouse, due to the link, therefore I'm not sure if it should be by keyboard 🤔

Reply:
@dhruvkb pinging you here again for visibility, thank you! I don't believe the box layout should be seekable.

Reply:
Yes, the box layout should not be seekable. It was considered bad UX to allow it to be seekable when the seeking area is super tiny and lies inside a link.

Reply:
Got it, thanks! So currently the box layout _is_ seekable, so as-written this code will work fine. I'll remove the comment and create a new issue to make the box layout not-seekable, at which point the labels should be updated.